lib/diagnostic.rb

```diff
@@ -108,7 +113,7 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'people{:favorite_number :first_name}'
```

Comment:
This would be correct:

`'person = {first_name: "Lauren", favorite_number: 7}'`